src/it/MCOMPILER-609/pom.xml

```diff
@@ -20,26 +20,29 @@ under the License.
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.example.pce</groupId>
-  <artifactId>sample</artifactId>
-  <version>0.0.1-SNAPSHOT</version>
+  <artifactId>MCOMPILER-609</artifactId>
+  <version>1.0-SNAPSHOT</version>
   <packaging>jar</packaging>
-  <properties>
-    <project.build.sourceEncoding>utf-8</project.build.sourceEncoding>
-  </properties>
+  <name>MCOMPILER-609</name>
+
   <build>
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>@pom.version@</version>
+        <version>@project.version@</version>
         <configuration>
-          <compilerId>eclipse</compilerId>
+          <compilerId>ecj</compilerId>
+          <!-- TODO: remove source and target after we identified where Maven inherits those values. -->
```

Comment:
Suggest to create an issue for this, so it's more visible

Reply:
Yes, the problem is that I have been unable to identify from where Maven inherits those values (`maven-core`? `maven-model-builder`? somewhere else?) NetBeans "effective" POM view does not show them.